CHANGELOG.md

```diff
@@ -21,6 +24,38 @@
 - This version upgrades Kubernetes to `v1.26`
 
 ### Added
+
+- Add additional pre-commit checks
+- Add new configuration file with compliantkubernetes-kubespray version
+- Dependency check to main bin scripts
+
+### Changed
+
+- Updated the Kubernetes audit log policy file
+- Added PSA labels for rook-ceph
+- Disabled Kubernetes PSPs for rook-ceph
+- Upgraded kubespray to v2.22.1
```

Comment:
This seems like a repeat? And the version says v2.22.1 here and under v2.21.0-ck8s3.